src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2409,6 +2409,17 @@ private static async Task<HttpStatusCode> GetGraphQLResponsePostConfigHydration(
             { entityName, entity }
         };
 
+            // Adding an entity with only Authorized Access
+            Entity anotherEntity = new(
```

Comment:
Why is this required?

Reply:
to generate a failure event while making an anonymous request to an Entity with authorized only access.